playbook.yml

```diff
@@ -117,6 +117,7 @@
       monit_protection: true
       nginx_install_method: package
       nginx_default_enable: false
+      nginx_client_max_body_size: "15M"
```

Comment:
Yeah, looks like the readme is out of date. Odd.

I'd vote for 5M. It's 7X the existing form sizes and is still bigger than any form I've ever seen. And if it goes over that amount we have this conversation to prove me wrong 😂